### PR TITLE
mo-shakib

### LINKED ISSUE #675: Mohammad Shakib
image

### PR DESCRIPTION
**Pull request by Mo-Shakib**
resolve #675

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please add your profile at the bottom.

File: data.json [hunk 1]
```diff
@@ -1288,11 +1288,17 @@
...
     "issueNumber": 666
+  },
+  {
+    "name": "Mohammad Shakib",
+    "githubUsername": "mo-shakib",
+    "imageUrl": "https://user-images.githubusercontent.com/50780268/131604251-f07f6811-d795-448f-a7b8-0f13b58c602f.png",
+    "issueNumber": 675
   }
-]
+]
```
Comment (Panquesito7) [inline data.json:1304]:
```suggestion
]

```

**Review by Panquesito7 — COMMENTED** [review]
Why close this pull request? 🤔

File: data.json (hunk 1 above)
Comment (Mo-Shakib) [reply data.json:1304]: Mistakenly I did that :3

**Pull request closed without merging**